pkg/codegen/python/gen.go

```diff
@@ -2210,6 +2222,21 @@ func genPackageMetadata(
 	return w.String(), nil
 }
 
+// errNoMinimumPythonVersion is a non-fatal error indicating that the schema
```

Comment:
Probably overkill for a `!= ""` check.

Reply:
Is this blocking feedback? I want a checked error because 
1. we've already missed checking this field once
2. we're about to check it in a second place, with the advent of the `pyproject.toml` file.

That's kind of the whole point of this PR, really.

Reply:
Thanks! I had a similar discussion with @abhinav around the API limitations which (in my eyes) necessitate adding this fn on a public field.